### PR TITLE
2322 Loans Database creation

### PR DESCRIPTION
**Pull request by kanjula-archana**
Loans Database Docker Compose , Data Load and Tables creation

### CODE REVIEW

File: devops/docker/docker-compose-db.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
+version: '3.8'
```
Comment (theqajedi) [inline devops/docker/docker-compose-db.yml:1]: I have deleted this file in my previous PR to unify the local build with bootstrap script.
Please add it to the main docker-compose file:
https://github.com/DeloitteDigitalAT/core-banking/blob/master/devops/docker/docker-compose.yml

Comment (kanjula-archana) [reply devops/docker/docker-compose-db.yml]: Removed this file and added code to the other

File: devops/docker/docker-compose.yml [hunk 2]
```diff
@@ -80,5 +80,18 @@ services:
     environment:
       - "SPRING_PROFILES_ACTIVE=test"
 
+  db2:
```
Comment (theqajedi) [inline devops/docker/docker-compose.yml:83]: If you don't mind, can you please add this before the services along with the other db's? 🙂 
Just to keep it orderly.

Comment (kanjula-archana) [reply devops/docker/docker-compose.yml]: Yup, Updated the order.

**Review by theqajedi — APPROVED** [review]

**Pull request merged**